src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -1555,6 +1555,8 @@ class device {
                 if (found != std::string::npos) {
                   auto scu_i = std::stoi(cu_n.substr(found + 4));
                   cu_n = cu_n.substr(0, found - 1);
+                  cu_n.append("_"); //Append instance number to name
+                  cu_n.append(std::to_string(scu_i));
```

Comment:
I am not sure if this is exactly the CR is requesting. Say, we have tow PS kernels and each one has 2 instances. What will be displayed here?
SK1_0
SK1_1
SK2_0
SK2_1
or
SK1_0
SK1_1
SK2_2
SK2_3